build/configs/rtl8730e/loadable_apps_16M/defconfig

```diff
@@ -463,11 +467,14 @@ CONFIG_GPIO=y
 # CONFIG_LCD is not set
 CONFIG_BCH=y
 CONFIG_RTC=y
+# CONFIG_RTC_DATETIME is not set
+# CONFIG_RTC_HIRES is not set
 # CONFIG_RTC_ALARM is not set
 CONFIG_RTC_DRIVER=y
-# CONFIG_RTC_HIRES is not set
 # CONFIG_RTC_IOCTL is not set
```

Comment:
Please enable CONFIG_RTC_IOCTL configuration

Reply:
Please let me know if you need these changes in loadable_apps only or all defconfigs?